examples/semantic_agent.py

```diff
@@ -57,8 +64,37 @@
         ],
     },
 ]
+
+
+
+
 agent = SemanticAgent([employees_df, salaries_df], config=config_, schema=schema)
 
 # Chat with the agent
-response = agent.chat("Plot a chart of the average salary of employees by department")
-print(response)
+# response = agent.chat("Plot a chart of the average salary of employees by department")
+# print(response)
+
+
+
+
+# Testing with a simple DataFrame
+df = pd.DataFrame(columns=["Empdata"], data=[[1], [2]])
+
+df.head()
+
+
+try:
+    #  Create an instance of the SemanticAgent with the provided dataframe
+    semantic_agent = SemanticAgent(dfs=df)
```

Comment:
This example need to reverted because it's not the valid use SemanticAgent we can definitely add comment about the custom schema if some one want to use different one but that too should be in documentation